# HG changeset patch
# User lo sppedflyer@gmail.com
# Date 1476962939 -7200
#      Thu Oct 20 13:28:59 2016 +0200
# Branch CMake
# Node ID c9e4435f36814864d1b6495ac3b665a0f2d24099
# Parent  73fc730619a6fa4b2047c2321e84b1bea53c185b
windows, another patch

diff -r 73fc730619a6 -r c9e4435f3681 src/CMakeLists.txt
--- a/src/CMakeLists.txt	Thu Oct 20 13:00:42 2016 +0200
+++ b/src/CMakeLists.txt	Thu Oct 20 13:28:59 2016 +0200
@@ -154,15 +154,15 @@
   ARCHIVE DESTINATION lib
 )
 
-if(MSVC)
-	if(COIN_BUILD_SHARED_LIBS)
-		install(FILES ${PROJECT_BINARY_DIR}/src/Debug/Coind.pdb DESTINATION bin CONFIGURATIONS Debug)
-		install(FILES ${PROJECT_BINARY_DIR}/src/RelWithDebInfo/Coin.pdb DESTINATION bin CONFIGURATIONS RelWithDebInfo)
-	else()
-		set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /Z7")                       # Override default /Zi to embed
-		set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /Z7")     # debugging info in the .lib.
-	endif()
-endif()
+# if(MSVC)
+# 	if(COIN_BUILD_SHARED_LIBS)
+# 		install(FILES ${PROJECT_BINARY_DIR}/src/Debug/Coind.pdb DESTINATION bin CONFIGURATIONS Debug)
+# 		install(FILES ${PROJECT_BINARY_DIR}/src/RelWithDebInfo/Coin.pdb DESTINATION bin CONFIGURATIONS RelWithDebInfo)
+# 	else()
+# 		set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /Z7")                       # Override default /Zi to embed
+# 		set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /Z7")     # debugging info in the .lib.
+# 	endif()
+# endif()
 
 # Support find_package(Coin NO_MODULE).
 set(COIN_INCLUDE_DIR include)
